cereus/src/main/java/com/cactus/entities/User.java

```diff
@@ -37,21 +50,33 @@ public String getUsername(){
     public String getPassword(){
         return password;
     }
-
-    public long getId() {
-        return this.id;
+    public String getToken(){
+        return token;
+    }
+    public long getId(){
+        return id;
     }
-
     public void setName(String name){
         this.name = name;
     }
-
+    public void setUsername(String username){
+        this.username = username;
+    }
+    public void setToken(String token){
+        this.token = token;
+    }
     public void setPassword(String password) {
         this.password = password;
     }
-
-    public void setId(long id) {
+    public void setId(long id){
         this.id = id;
     }
-
+    @Override
+    public String toString(){
+        return "Name: " + name +
+                "\nUsername: " + username +
+                "\nPassword: " + password +
+                "\nToken: " + token +
+                "\nId: " + id;
+    }
```

Comment:
Do we need toString methods?

Reply:
I was using this for testing, so I'll just delete the method.